src/component/employee/index.js

```diff
@@ -0,0 +1,61 @@
+import './employee.scss'
+import React from 'react'
+import {connect} from 'react-redux'
+import EmployeeItem from '../employee-item'
+import EmployeeForm from '../employee-form'
+import * as employee from '../../action/employee.js'
+
```

Comment:
Your components are all structured very well! They have thoughtful names and very readable